pkg/controller/cassandrabackup/cassandrabackup_controller.go

```diff
@@ -47,23 +108,28 @@ func (r *ReconcileCassandraBackup) Reconcile(request reconcile.Request) (reconci
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling CassandraBackup")
 
-	// Get request params
-	b := &cassandraoperatorv1alpha1.CassandraBackup{}
-	if err := r.client.Get(context.TODO(), request.NamespacedName, b); err != nil {
+	// Fetch the CassandraBackup instance
+	instance := &cassandraoperatorv1alpha1.CassandraBackup{}
```

Comment:
can we call this `backup` instead of `instance`? "Instance" is very ambiguous, and has a lot of meanings, which can be very confusing in a backup context.

Reply:
`instance` comes from the SDK scaffolding code generator. True, it's generic, probably on purpose. I guess the SDK folks wanted `instance` to always mean "the resource which needs reconciling" if that makes sense.

We can change it to `backup` or back to `b`, however this would make it slightly harder to figure out the next time we do an SDK version upgrade. WDYT?

Reply:
I would leave it to be `instance` in this particular case.